packages/react/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@clr/react",
+  "version": "0.0.0",
+  "scripts": {
+    "start": "npm-run-all core:build dev:start",
+    "dev:start": "parcel index.html",
+    "build": "npm-run-all -s build:ts build:package build:clean build:copy",
+    "build:clean": "node ./package.js",
+    "build:copy": "cpy './react' '../../../dist' --cwd=./dist --parents",
+    "build:package": "cpy '**/package.json' '../dist/react/' --cwd=./src --parents && cpy ./package.json ./dist/react/ && cpy ./README.md ./dist/react/; replace '0.0.0' $npm_package_version ./dist/react/package.json",
+    "build:ts": "tsc --b ./tsconfig.project.json --force",
+    "core:build": "yarn --cwd packages/core run build",
+    "lint": "eslint \"*/**/*.{ts,tsx}\" --ignore-pattern \"dist\"",
+    "lint:fix": "eslint --fix \"*/**/*.{ts,tsx}\" --ignore-pattern \"dist\"",
+    "test": "npm-run-all core:build test:coverage",
```

Comment:
Since this is called within the main test script we wont want to build core as it would cause it to be built twice. Also the tests are run in parallel so this could cause issues of building while core is attempting to run its unit tests.

Reply:
I will remove the build step here and just have `test` script here run jest with code coverage.